packages/form-handlers-mailchannels-test/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import { mailchannelsHandler } from "@webstudio-is/form-handlers";
+
+export default {
+  async fetch(request: Request) {
+    const url = new URL(request.url);
+
+    const to = url.searchParams.get("to");
+
+    if (to == null) {
+      return new Response(
+        `Set "to" query parameter to your email. Example: https://...?to=bob@example.com`
+      );
+    }
+
+    const formData = new FormData();
+    formData.append("name", "Bob");
+    formData.append("time", new Date().toLocaleString("en-US"));
+
+    const result = await mailchannelsHandler({
+      senderEmail: "test@webstudio.is",
```

Comment:
mb `@example.com` ?

Reply:
Mailchanels doesn't allow to send from `example.com` 🤷‍♂️